### ISSUE TITLE
Groomsmen's Gifts

### ISSUE THREAD

**Issue by shayanik** [shayanik/Shayannon-Wedding#20]
Default: Watches

Thinking of getting everyone a unique vintage russian watch. Not sure though

**Comment by mihorhino** [comment]
Hm, what do you think about engraved whiskey bottles?

Or engraved decanter's (since Kevin doesn't drink).

My only concern about the Russian watches from Etsy is the quality isn't great. And they style of those watches isn't for everyone (though you could say that about everything).

**Comment by shayanik** [comment]
I'm kinda liking the decanter idea. Only person I wonder about is Kevin. He doesn't drink. I wonder if he could find value in it. Does wine keep? No, right?

**Comment by mihorhino** [comment]
Yeah, maybe Kevin wouldn't find value in it (other than it's decorative value).

Other ideas:

Engraved cuff links
Personalized pocket watches
